articles/rules/current/index.md

```diff
@@ -13,32 +13,44 @@ useCase:
 ---
 # Rules
 
-**Rules** are functions written in JavaScript that are executed when a user authenticates to your application. They run once the authentication process is complete and you can use them to customize and extend Auth0's capabilities. They can be chained together for modular coding and can be turned on and off individually.
+**Rules** are functions written in JavaScript that are executed when a user authenticates to your application. They run once the authentication process is complete, and you can use them to customize and extend Auth0's capabilities. They can be chained together for modular coding and can be turned on and off individually.
 
 ![Rule Flow](/media/articles/rules/flow.png)
 
 1. An app initiates an authentication request to Auth0.
 1. Auth0 routes the request to an Identity Provider through a configured connection.
 1. The user authenticates successfully.
-1. The tokens ([ID Token](/tokens/id-token) and/or [Access Token](/tokens/access-token)) pass through the Rules pipeline, and are sent to the app.
+1. The tokens ([ID Token](/tokens/id-token) and/or [Access Token](/tokens/access-token)) pass through the Rules pipeline and are sent to the app.
 
-## What can I use rules for?
+## What can I use Rules for?
 
 Among many possibilities, rules can be used to:
 
-* __Profile enrichment__: query for information on the user from a database/API, and add it to the user profile object.
+* __Enrich user profiles__: query for information on the user from a database/API, and add it to the user profile object.
 * Create __authorization rules__ based on complex logic (anything that can be written in JavaScript).
 * __Normalize attributes__ from different providers beyond what is provided by Auth0.
 * Reuse information from existing databases or APIs for migration scenarios.
 * Keep a __white-list of users__ and deny access based on email.
 * __Notify__ other systems through an API when a login happens in real-time.
 * Enable counters or persist other information. For information on storing user data, see: [Metadata in Rules](/rules/metadata-in-rules).
-* Enable __multifactor__ authentication, based on context (such as last login, IP address of the user, location, and so on).
+* Enable __multi-factor authentication__, based on context (such as last login, IP address of the user, location, and so on).
 * Modify tokens: Change the returned __scopes__ of the Access Token and/or add claims to it, and to the ID Token.
 
-## Video: Using rules
+---
+
+<img align="right" src="/media/articles/rules/extend-rules.svg"> 
+
+## Auth0 Rules in your SaaS
+
+If you want to create and customize Rules in your product, try Auth0 Extend--an embedded scripting environment for enabling customization and integration of your SaaS.
+
+[Try Extend &#9658;](https://goextend.io/)
```

Comment:
Please use `https://goextend.io?utm_source=docs&utm_medium=page&utm_campaign=auth0-com&utm_content=rules-current-paragraph` for URL

Reply:
Done.